docs/content/docs/recipes/angular.md

```diff
@@ -1,6 +1,276 @@
+
+### Get the Fluid Data
+
+1. The Fluid runtime will bring changes made to the timestamp from any client to the current client. But Fluid is agnostic about the UI framework. You can use a helper method to get the Fluid data, from the SharedMap object, into the component layer. Add the following code inside the `AppComponent` class. This function will be called when the application loads the first time, and the value it returns is assigned to the `sharedTimestamp` component level property.
+
+    ```js
+    async getFluidData() {
+
+      // TODO 1: Configure the container.
+      // TODO 2: Get the container from the Fluid service.
+      // TODO 3: Return the Fluid timestamp object.
+    }
+    ```
+
+1. Replace `TODO 1` with the following code. Note that there is only one object in the container: a SharedMap holding the timestamp. Note also that `sharedTimestamp` is the ID of the `SharedMap` object and it must be unique within the container.
+
+    ```js
+    const client = new TinyliciousClient();
+    const containerSchema = {
+      initialObjects: { sharedTimestamp: SharedMap }
+    };
+    ```
+
+1. Replace `TODO 2` with the following code. Note that `containerId` is being stored on the URL hash, and if there is no `containerId` we create a new container instead.
+
+    ```js
+    let container;
+    const containerId = location.hash.substring(1);
+    if (!containerId) {
+      ({ container } = await client.createContainer(containerSchema));
+      const id = await container.attach();
+      location.hash = id;
+    }
+    else {
+      ({ container } = await client.getContainer(containerId, containerSchema));
+    }
+    ```
+
+1. Replace `TODO 3` with the following code.
+
+    ```js
+    return container.initialObjects.sharedTimestamp as SharedMap;
+    ```
+
+### Keep the Fluid data synchronized
+
+To ensure that both local and remote changes to the timestamp are reflected in the UI, we will use the `localTimestamp` component property to store the local timestamp value and ensure that it is updated whenever any client changes the `fluidSharedObjects` value.
+
+1. Below the preceding `getFluidData` function add the following code.
+
+    ```js
+    syncData() {
+      // Only sync if the Fluid SharedMap object is defined.
+      if (this.sharedTimestamp) {
+        // TODO 4: Set the value of the localTimestamp object that will appear in the UI.
+
+        // TODO 5: Register handlers.
+      }
+    }
+    ```
+
+1. Replace `TODO 4` with the following code. Note about this code:
+
+    - The Fluid `SharedObject.get` method returns the data of the `SharedObject` (in this case the `SharedMap` object), which is roughly the `SharedObject` without any of its methods. So the `updateLocalTimestamp` function is setting the `localTimestamp` property to a copy of the data of the `SharedMap` object. (The key "time" that is passed to `SharedObject.get` is created in a later step. It will have been set by the time this code runs the first time.)
```

Comment:
"shared object" is not a specific thing but instead a generic term that describes distributed data structures. Not all DDSes/SharedObjects are the same and they can expose any API that enables their scenario. 

```suggestion
    - `this.sharedTimestamp` is an instance of a `ShareMap` which exposes the ability to set/get from the API. The `updateLocalTimestamp` function is setting the `localTimestamp` property to the value of the key `"time"` on the `sharedTimestamp`. (The "time"key is created in a later step. It will have been set by the time this code runs the first time.)
```